core/src/options/input_field.rs

```diff
@@ -97,11 +98,7 @@ impl InputField {
         // 1. Will we look for this field in the attribute?
         // 1. Is there a locally-defined default?
         // 1. Did the parent define a default?
-        self.default = match (
-            self.skip.unwrap_or_default(),
-            self.default.is_some(),
-            parent.default.is_some(),
-        ) {
+        self.default = match (&self.skip, self.default.is_some(), parent.default.is_some()) {
```

Comment:
An NFC to clean this up seems sensible:

- Make the first case be a simple if-statement, since it overrules the others
- Possibly rework second and third variables as well?